apps/server/src/core/services/sqlite-service.ts

```diff
@@ -76,8 +72,17 @@ export class SQLiteService {
   }
 
   static async getMatchingContext(
-    embedding: number[]
+    embedding: number[],
+    threshold: string | undefined,
+    count: string | undefined,
+    minLength: string | undefined
```

Comment:
These can actually be 
```suggestion
    threshold?: string,
    count?: string,
    minLength?: string
```